Mathlib/Algebra/TrivSqZeroExt.lean

```diff
@@ -936,6 +936,83 @@ def liftEquivOfComm :
       rfl }
 #align triv_sq_zero_ext.lift TrivSqZeroExt.liftEquiv
 
+variable {N P : Type*} [AddCommMonoid N] [Module R' N] [Module R'ᵐᵒᵖ N] [IsCentralScalar R' N]
```

Comment:
I think it would be reasonable to add `section map` around all of these new results.